content/en/tracing/setup_overview/custom_instrumentation/dotnet.md

```diff
@@ -119,9 +119,59 @@ using (var parentScope =
     }
 }
 ```
+
+### Headers extraction and injection
+
+Datadog APM tracer supports [B3][5] and [W3C][6] headers extraction and injection for distributed tracing. You can find more details on this in the [setup documentation][7].
```

Comment:
```suggestion
The Datadog APM Tracer supports [B3][5] and [W3C][6] headers extraction and injection for distributed tracing. For more information, see the [setup documentation][7].
```

Reply:
Including the change. But note that this text was a copy paste from the Java and Go do IIRC.